lib/models/src/main/java/org/sagebionetworks/repo/model/NodeDAO.java

```diff
@@ -79,6 +79,14 @@
 	public boolean delete(String id) throws DatastoreException;
 	
 	/**
+	 * Delete all nodes within a list of IDs.

```

Comment:
Delete all nodes within a list of IDs and no nodes has more than 15 level of children